src/scenechange/mod.rs

```diff
@@ -99,11 +73,20 @@ impl SceneChangeDetector {
       return true;
     }
 
-    // Skip smart scene detection if it's disabled
     if config.speed_settings.no_scene_detection {
       return false;
     }
 
+    self.exclude_scene_flashes(&frame_set, input_frameno, inter_cfg);
```

Comment:
I feel like I had a reason for running this on every frame, although logically it feels unnecessary. I'm going to throw this branch in AWCY just as a sanity test.

Reply:
Thank you :)